src/db/sql/sqlSelectBuilder.ts

```diff
@@ -5,45 +5,45 @@ import { Table } from '../table'
 import { SqlBuilder } from './sqlBuilder'
 
 export class SqlSelectBuilder extends SqlBuilder {
-  private _select: Array<Column> = []
+  private _select: Array<Column | string> = []
   private _from: Array<Table> = []
   private _where: Array<string> = []
   private _groupBy: Array<Column> = []
   private _offset: number | null = null
   private _limit: number | null = null
   private _orderBy: Array<Column> = []
 
-  select(...fields: Array<Column>): SqlSelectBuilder {
+  select(...fields: Array<Column | string>): this {
```

Comment:
if we want to be consistent, we should allow Column or string even in the other methods.

Reply:
what methods and why?

Reply:
I mean even orderBy etc, but we can change them later, if needed.